pkg/fixtures/pod.go

```diff
@@ -112,6 +113,222 @@ func GetPod() *storage.Pod {
 	}
 }
 
+// GetPod1 returns a more realistic mock Pod than GetPod
+func GetPod1() *storage.Pod {
+	return &storage.Pod{
+		Id:	      fixtureconsts.PodUID1,
```

Comment:
In the old GetPod the Id is not a valid Uid. This means that it cannot be inserted into a postgres database. It seems that this might not have been a problem with other unit tests, because they mock somethings. Not 100% sure about this. Also GetPod did not have a name for the pod, so a pod name is set here.